frontend/packages/pipelines-plugin/locales/en/pipelines-plugin.json

```diff
@@ -3,6 +3,9 @@
   "Create a Tekton Pipeline to automate delivery of your Application": "Create a Tekton Pipeline to automate delivery of your Application",
   "Tasks": "Tasks",
   "Triggers": "Triggers",
+  "Tekton": "Tekton",
```

Comment:
Translations have a lot of `Tekton` references. Not sure we want this (per my previous comment)